core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -96,6 +98,45 @@ case class LogConfig(props: java.util.Map[_, _], overriddenConfigs: Set[String]
   val LeaderReplicationThrottledReplicas = getList(LogConfig.LeaderReplicationThrottledReplicasProp)
   val FollowerReplicationThrottledReplicas = getList(LogConfig.FollowerReplicationThrottledReplicasProp)
   val messageDownConversionEnable = getBoolean(LogConfig.MessageDownConversionEnableProp)
+  val remoteStorageEnable = getBoolean(LogConfig.RemoteLogStorageEnableProp)
```

Comment:
Why do we use `remoteStorageEnable` but are using `RemoteLogStorageEnable` in the following place
https://github.com/satishd/kafka/blob/6e66695e87c8dd7db33cb134e83e0281e2216a62/core/src/main/scala/kafka/log/LogConfig.scala#L43

Should we rename is to `remoteLogStorageEnable`?